consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/queries/IbftQueryServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.ibft.queries;
+
+import org.hyperledger.besu.consensus.common.BlockInterface;
+import org.hyperledger.besu.consensus.common.PoaQueryServiceImpl;
+import org.hyperledger.besu.consensus.ibft.IbftBlockHashing;
+import org.hyperledger.besu.consensus.ibft.IbftExtraData;
+import org.hyperledger.besu.ethereum.chain.Blockchain;
+import org.hyperledger.besu.ethereum.core.BlockHeader;
+import org.hyperledger.besu.ethereum.core.Hash;
+import org.hyperledger.besu.plugin.data.Address;
+import org.hyperledger.besu.plugin.services.metrics.PoAMetricsService;
+import org.hyperledger.besu.plugin.services.query.IbftQueryService;
+import org.hyperledger.besu.util.bytes.Bytes32;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public class IbftQueryServiceImpl extends PoaQueryServiceImpl
+    implements IbftQueryService, PoAMetricsService {
```

Comment:
nit: You don't need to have `PoAMetricsService` in the implements list here because `PoaQueryServiceImpl` already implements it.
```suggestion
    implements IbftQueryService {
```

Reply:
done